vale/fixtures/Grafana/Kubernetes/testinvalid.md

```diff
@@ -0,0 +1,10 @@
+Don't use:
+
+- cron job
+- deamonset
+- Kubernetes deployment
+- pod
+- replicaset
+- statefulset
+- kubelet
+- kubectl
```

Comment:
Do we need tests for Kubectl and Kubelet (with initial caps)?

Reply:
Good catch, added in [2f9a383](https://github.com/grafana/writers-toolkit/pull/512/commits/2f9a3839edf1bc75362eb3e03774ad1d1293e0d2)